databricks/koalas/plot.py

```diff
@@ -1148,6 +1215,10 @@ def area(self, **kwds):
 
         Examples
         --------
+
+        .. plot::
+            :context: close-figs
+
         >>> df = ks.DataFrame({
```

Comment:
indentation seems inconsistent too

Reply:
Thanks for comment!! fixed both of them.